app/views/todos.ejs

```diff
@@ -12,20 +12,32 @@
 </head>
 
 <body>
-    <div class="app">
-        <header>
-            <nav class="navbar navbar-dark bg-primary">
-                <span class="navbar-brand mb-0 h1">Todo App</span>
-            </nav>
-        </header>
-        <div class="container">
-            <div class="row">
-                <div class="col-md-8 m-auto pt-4">
+    <div class="container m-5 p-2 rounded mx-auto bg-light shadow">
+        
+        <!-- App title section -->
+        <div class="row m-1 p-4">
+            <div class="col">
+                <div class="p-1 h1 text-primary text-center mx-auto display-inline-block">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" width="50" height="50">
+                        <!-- !Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc. -->
```

Comment:
Hello, thank you so much for contributing! We appreciate it. Would it be possible to take out the logo and change the `todo's` to `Todo App`

Reply:
HI @sfavello  sure i can make the necessary changes, I added the docker logo for brand sake but I can make the change and also thanks for reviewing the code.